### ISSUE TITLE
[Feature Request] Release for Windows IoT (RPi)

### ISSUE THREAD

**Issue by scegg** [SoftEtherVPN/SoftEtherVPN#964]
Hi.

I'm developing some network based device using Raspberry Pi. Currently, I'm using it based on Linux and coding with dotNet Core 2.
When trying to operate bluetooth or some device based item with dotnet core 2 on Linux, it is not fully supported. The only reason keep me using Linux is the SoftEther cannot support Windows IoT version.

Could you make a release build on Windows IoT? Thanks.

**Comment by davidebeatrici** [comment]
Hi,

In theory we only have to build binaries for ARM64, right?

**Comment by chipitsine** [comment]
Nope. Win on arm64 requires ndis 6.3 certified drivers. It was a pain for OpenVPN
